core/components/molecules/page-header/page-header.js

```diff
@@ -40,7 +40,9 @@ const PageHeader = props => {
         )}
       </ButtonGroup>
 
-      <Heading size={1}>{props.title}</Heading>
+      <Heading size={1} margin={{ top: 0, bottom: 'xsmall' }}>
```

Comment:
I suggest we merge this https://github.com/auth0/cosmos/pull/1138 before we move on with margin. Since it changes the way we do spaces on the page header

Reply:
Sure thing! added that as a blocker to merge in the description